test/Sandpit/Exscientia/Free_Energy/test_relative.py

```diff
@@ -0,0 +1,59 @@
+import pathlib
+
+import pytest
+import pandas as pd
+import numpy as np
+from alchemlyb.parsing.gmx import extract_u_nk
```

Comment:
`alchemlyb` isn't part of the development BioSimSpace dependencies, so this would break our CI. Please wrap the import in a `try/except` block and skip the tests if it can't be imported.

Reply:
Because it is currently not used by `devel`. It will be added when this functionality is moved out of the sandpit. I think it's only just gone on conda-forge, which is great news! (If we want to move there then we can't use any pip packages or vendor any code.)